src/vivarium_gates_nutrition_optimization/components/disease.py

```diff
@@ -40,7 +46,7 @@ def __init__(
         side_effect_function : callable, optional
             A function to be called when this state is entered.
         """
-        super().__init__(cause, **kwargs)
+        super(DiseaseState_, self).__init__(cause, **kwargs)
```

Comment:
Just to make sure I'm understanding this correctly, this effectively means calling the `super()` method of `DiseaseState_`, or alternately could have been written as `super().super().__init__()`?